binding/binding_test.go

```diff
@@ -3,68 +3,46 @@ package binding
 import (
 	"net/http"
 	"testing"
-	"time"
 
 	"github.com/stretchr/testify/require"
 )
 
 func Test_Register(t *testing.T) {
 	r := require.New(t)
-	l := len(binders)
+	l := len(defaultRequestBinder.binders)
 	Register("foo/bar", func(*http.Request, interface{}) error {
 		return nil
 	})
-	r.Len(binders, l+1)
-}
 
-func TestParseTimeErrorParsing(t *testing.T) {
-	r := require.New(t)
-	_, err := parseTime([]string{"this is sparta"})
-	r.Error(err)
+	r.Len(defaultRequestBinder.binders, l+1)
 }
 
-func TestParseTime(t *testing.T) {
+func Test_RegisterCustomDecoder(t *testing.T) {
+	// r := require.New(t)
 
-	r := require.New(t)
+	// RegisterCustomDecoder(func(vals []string) (interface{}, error) {
```

Comment:
lots of commented out code kicking about

Reply:
This one still pending I'll cover it soon.